### ISSUE TITLE
[Suggestion] Teleporthit

### ISSUE THREAD

**Issue by The-Warlord** [MeteorDevelopment/meteor-client#646]
Inertia and LiquidBounce have this module, which allows you to hit at distances of >6 blocks. LiquidBounce's tphit has unlimited range, while Inertia only has 10 block range. Not sure if this is because of limitations in 1.16, but it would be pog if Meteor had teleporthit so I can uninstall Inertia.

**Comment by Declipsonator** [comment]
What servers does this even work on 🤔

**Comment by AutistEren** [comment]
in inertia you can type .value reach range 50 smh

**Comment by Wide-Cat** [comment]
Pretty sure 1.16 has limitations on how far you can tp in a single tick or smthn so I don't think unlimited range tphit is possible. Does the inertia 10 block one work on 1.16?

**Comment by The-Warlord** [comment]
> What servers does this even work on 🤔

Vanilla servers


> in inertia you can type .value reach range 50 smh
Thank you, didn't know you could set values like that.
> PLUS you can just use inertia and meteor together
I am using both at the same time, but tphit is literally the only thing I have inertia for. It would be nice to not have to use Inertia.

**Comment by The-Warlord** [comment]
> Pretty sure 1.16 has limitations on how far you can tp in a single tick or smthn so I don't think unlimited range tphit is possible. Does the inertia 10 block one work on 1.16?

Yes, 10 block reach does work. Doesn't work for anything higher than that, unfortunately.

**Comment by ManInMyVan** [comment]
> Maybe a bypass is possible?

Yes #3825 is one (for paper servers)

**Comment by Fiz-Victor** [comment]
I have created an addon and it has a module called "LOReach", you can use that
